docs/connectors/kafka-connect-wrapper/overview.mdx

```diff
@@ -0,0 +1,158 @@
+
+### Pre-requisites
+
+* JDK 11
+* Currently, only Unix-like OSes are supported.
+
+### Getting Started
+
+The general process follows these steps:
+1. Build the Conduit Kafka Connect Wrapper.
+2. Download the Kafka Connect JAR you want to use plus all of its dependencies
+3. Drop in newly downloaded Kafka Connect Connnectors into the `libs` directory
+4. Create connection in Conduit with your Kafka Connect configuration
+
+#### Building and using the wrapper
+
+Start by cloning thee repository, [conduit-kafka-connect-wrapper](https://github.com/ConduitIO/conduit-kafka-connect-wrapper).
+
+```
+git clone git@github.com:ConduitIO/conduit-kafka-connect-wrapper.git
+```
+
+Run `scripts/dist.sh` in the newly clone repository to build an executable. `scripts/dist.sh` will create a directory called `dist` with following contents:
+1. A script (which runs the connector). This script starts a connector instance.
+2. The connector JAR itself
+3. Directory `libs`. This is where you put the Kafka Connect Connector JARs and their dependencies (if any).
+
+When creating a Conduit connector, the plugin path you need to use is the path to `conduit-kafka-connect-wrapper`. Here's a full working example of a Conduit connector configuration:
+```json
+{
+  "type": "TYPE_SOURCE",
+  "plugin": "/home/conduit-dev/projects/conduitio/conduit-kafka-connect-wrapper/dist/conduit-kafka-connect-wrapper",
+  "pipeline_id": "f24c4a70-8664-4f80-9c27-204825442943",
+  "config": {
+    "name": "my-pg-source",
+    "settings": {
+      "wrapper.connector.class": "io.aiven.connect.jdbc.JdbcSourceConnector",
+      "connection.url": "jdbc:postgresql://localhost/conduit-test-db",
+      "connection.user": "username",
+      "connection.password": "password",
+      "incrementing.column.name": "id",
+      "mode": "incrementing",
+      "tables": "customers",
+      "topic.prefix": "my_topic_prefix"
+    }
+  }
+}
+```
+
+Note that the `wrapper.connector.class` should be a class which is present on the classpath, i.e. in one of the JARs in
+the `libs` directory. For more information, theck the [Configuration](#configuration) section.
+
+### Loading Kafka Connect connectors
+All Kafka Connect Connectors will load, along with all the other dependencies, from a `libs` directory, which is expected to be in the
+same directory as the Conduit Connector executable itself. For example, if the Conduit Connector executable is at `/abc/def/conduit-kafka-connect-wrapper`,
+then the dependencies are expected to be in `/abc/def/libs`.
+
+The plugin will be able to find the dependencies as soon as they are put into `libs`. Please note that, a JDBC connector  (generally) will require a database-specific driver to work (for example, PostgreSQL's driver can be found [here](https://mvnrepository.com/artifact/org.postgresql/postgresql)).
+
+
+#### Configuration
```

Comment:
I believe it's very useful having this configuration here, in the sense that it's easy for the user to quickly check what's available and what not. On the other hand, this means that we have to keep it in sync with the ReadMe in the wrapper repo. Can we somehow load it from the repo?